falco.yaml

```diff
@@ -544,3 +544,84 @@ metadata_download:
 base_syscalls:
   repair: false
   custom_set: []
+
+# stats_v2: periodic stats and resource utilization metrics, initial release
+#
+# --- [Description]
+#
+# stats_v2 reflects a stats re-design from the ground up. Falco now natively supports
+# resource utilization metrics and enhanced specialized metrics to monitor Falco's
+# performance in production. The new metrics are exposed as monotonic counters or snapshots
+# emitted at a pre-defined interval. Each metric is captured in the same consolidated log message.
+# In addition, relevant wrapper fields are added, allowing you to perform sophisticated and customized
+# statistical analyses and correlations. Lastly, the new stats framework can be easily extended
+# with new metrics.
+#
+# Notice: Schema and output field names are not guaranteed to be stable
+# and might change until stats_v2 reaches a stable release.
+#
+# --- [Usage]
+#
+# Disabled by default.
+#
+# Env variable `-E FALCO_HOSTNAME=myhostname` customizes hostname in case
+# you deploy in Kubernetes where the hostname can be equivalent to the pod name.
+#
+# `stats_interval_preset`:
```

Comment:
Lovely suggestions, let's go with them. It makes sense especially because we want to also offer a Prometheus integration once the new metrics become stable for Falco 0.36.